docs/tutorial.rst

```diff
@@ -394,3 +399,176 @@ estimates were obtained when Asian samples were compared with themselves.
+
+    >>> ts0 = ts.simplify(samples=pop0_samples)
+    >>> ts1 = ts.simplify(samples=pop1_samples)
+
+We are now ready to calculate the SFS.
+Since our dataset was generated using the default ``msprime`` simulation engine,
+we know that it has exactly one derived allele at any polymorphic site.
+We also know what the derived and ancestral states are.
+We can therefore calculate the *polarised* SFS using tskit's inbuilt
+:meth:`tskit.TreeSequence.allele_frequency_spectrum` method:
+
+.. code-block:: python
+
+    >>> sfs0 = ts0.allele_frequency_spectrum(polarised=True,span_normalise=False)
+    >>> print(sfs0)
+    [   0. 1165.  391.  232.  120.  117.   80.   91.   54.   55.    0.]
+
+.. code-block:: python
+
+    >>> sfs1 = ts1.allele_frequency_spectrum(polarised=True,span_normalise=False)
+    >>> print(sfs1)
+    [  0. 510. 251. 177. 159.  98.  72.  73.  73.  83.   0.]
+
+(Note: the ``polarised=True`` option indicates that we wish to calculate the statistic
+for derived alleles only, and the ``span_normalise=False`` option disables tskit's
+default behaviour of dividing by the sequence length.)
+
+We will do further analysis in the next section, but you might first wish to convince
+yourself that this output makes sense to you.
+We can see that amongst the mutated alleles held in population 1, there are 510
+singletons, 251 doubletons and so on.
+You might also wish to check that the total sum of sites is the sum of the SFS entries:
+
+.. note::
+    
+    why isn't it???
+
+.. code-block:: python
+
+    >>> sum(sfs1)
+    1068.0
+    >>> ts1.num_sites
+    1121
+
+----------------------
+3. Plotting the SFS
+----------------------
+
+Here is some more advanced code that compares the estimated SFS from each population.
+It relies on the ``matplotlib`` and ``numpy`` packages.
+We will scale each SFS by the number of mutated sites in the corresponding sample set.
+
+.. code-block:: python
+
+    >>> import matplotlib.pyplot as plt
+    >>> import numpy as np
+    >>> bar_width = 0.4
+    >>> r1 = np.arange(0,11) - 0.2
+    >>> r2 = [x + bar_width for x in r1]
+    >>> ax = plt.subplot()
+    >>> plt.bar(x = r1, height = sfs0/ts0.num_sites, width=bar_width)
+    >>> plt.bar(x = r2, height = sfs1/ts1.num_sites, width=bar_width)
+    >>> plt.xlabel("Allele count", fontweight="bold")
+    >>> plt.ylabel("Proportion of mutated sites in sample", fontweight="bold")
+    >>> ax.set_xticks(np.arange(0,11))
+    >>> ax.legend(['pop0', 'pop1'])
+    >>> plt.plot()
+
+.. image:: _static/tute-sfs.png
+    :align: center
+    :alt: SFS plots.
+
+This figure shows that population 0 contains a higher proportion of singletons than population 1, but that alleles of any other frequency are relatively more common in population 1.
+This disparity in proportion increases with the frequency of the allele.
+This makes sense given the demography we have specified;
+since population 1 is much smaller, it is subject to higher rates of genetic drift.
```

Comment:
Hm - I think this explanation is not quite right, since the expected normalized AFS does not depend on the size of a single randomly mating population. The difference between the two must have to do with this demographic history (somehow? haven't thought it through).

Reply:
I think it has to do with that population 1 had more extreme population size reduction than population 0. So, population 0 is closer to a neutral expectation. I think the reduction in population size in population 1 causes the population to loose rare alleles.